### PR TITLE
Added windows to docs.yml

### PR DESCRIPTION
**Pull request by HenryKobin**
Added windows to list of docs in docs.yml after adding updated tutorial to the windows page (#6100)

/cc @jekyll/documentation

### CODE REVIEW

File: docs/_data/docs.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@
   - usage
   - structure
   - configuration
+  - windows
```
Comment (DirtyF) [inline docs/_data/docs.yml]: We could keep the `Jekyll on Windows` title and put it under the installation entry

**Comment by HenryKobin** [comment]
@DirtyF I think it would be fine under Installation instead of Windows, because I'm running it with Ruby 2.3 / Jekyll and it's basically running as if I had it natively on Linux. I feel like Bash on windows is reaching the point where we could start allowing Windows users to feel more like they are an officially supported language, so they can at least get involved and start troubleshooting for themselves / contributing more to the jekyll ecosystem.
cc/ @jekyll/documentation @pathawks  @jekyllbot

File: docs/_data/docs.yml (hunk 1 above)
Comment (DirtyF) [inline docs/_data/docs.yml]: this entry should be next to installation

**Comment by DirtyF** [comment]
@HenryKobin I was only talking about the order of the links in the navigation menu. Let's keep it as a separate page for now. If @jekyll/windows says it's bug free, then we'll think about merging all installation instructions on the very same page.

**Comment by HenryKobin** [comment]
@DirtyF corrected.

**Comment by DirtyF** [comment]
@jekyllbot: merge +docs

**Pull request merged**